jobs/utils.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper functions for beam validators and one-off jobs"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+
+def clone_model(model, **new_values):
```

Comment:
Do you think it would be OK to have this as a function in base_model_validator (possibly as a method in the base class so that the individual validators can call self._clone_model())? Having it in a utility file is likely to encourage more use of it, which I think we want to avoid.

Reply:
/cc @brianrodri @vojtechjelinek too

Reply:
We originally had it as a function of `BaseModelValidator` but we also need to clone models for one-off jobs and didn't want to duplicate code. @brianrodri suggested putting it in a utils file

Reply:
Ah ok, sounds good then. Maybe clarify in the fileoverview which things the job_utils file is meant for. Thanks!